scripts/examples/gn_efr32_example.sh

```diff
@@ -94,7 +94,10 @@ if [ "$#" == "0" ]; then
         Presets
         --sed
             enable sleepy end device, set thread mtd
-            For minimum consumption, disable openthread cli and qr code
+            For minimum consumption, add --low-power
+        --low-power
```

Comment:
please change build_examples.py as well (targets.py and efr32.py specifically).

Ideally we should not have 2 separate scripts.

Reply:
Added the flags to the efr script as well